mycore-mods/src/main/resources/xsl/mycoreobject-datacite.xsl

```diff
@@ -193,9 +193,18 @@
       <xsl:when test="mods:originInfo[not(@eventType) or @eventType='publication']/mods:dateIssued[@encoding='marc']">
         <xsl:apply-templates select="mods:originInfo[not(@eventType) or @eventType='publication']/mods:dateIssued[@encoding='marc']" mode="publicationYear" />
       </xsl:when>
-      <xsl:otherwise>
+      <xsl:when test="mods:originInfo[not(@eventType) or @eventType='publication']/mods:dateCreated">
         <xsl:apply-templates select="mods:originInfo[not(@eventType) or @eventType='publication']/mods:dateCreated" mode="publicationYear" />
-      </xsl:otherwise>
+      </xsl:when>
+      <xsl:when test="mods:relatedItem/mods:originInfo[not(@eventType) or @eventType='publication']/mods:dateIssued[@encoding='w3cdtf']">
```

Comment:
type of relatedItem seems to be important to me, like only from "host" or similar